src/execution_plan.c

```diff
@@ -443,6 +447,16 @@ static const char* FlatExecutionPlan_SerializeInternal(FlatExecutionPlan* fep, s
 }
 
 int FlatExecutionPlan_Serialize(Gears_BufferWriter* bw, FlatExecutionPlan* fep, char** err){
+    
```

Comment:
You need to put it in the FlatExecutionPlan_Distributed, this function is also used by rdb save and on rdb save we do want to save the flat execution ..